addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -268,6 +268,13 @@ export class Dropdown extends Component {
         }
     }
 
+    /**
+     * Return if it's a different ui active element
+     */
+    isInActiveElement() {
+        return this.ui.activeElement !== this.myActiveEl;
```

Comment:
Given the function name it should be `this.ui.activeElement === this.myActiveEl`

Reply:
Done changes :+1: